css/rjsf.css

```diff
@@ -1,25 +1,25 @@
-.generic-form * {
+.rjsf * {
```

Comment:
is `rjsf` a fake/random name just for the tests, as I believe it is? If so, I'd rather have something like `random-name` or `test-name` or even `foobar` so it's explicit ;)